Content.Client/Chat/MarkupTags/EntityNameHeaderContentTag.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Diagnostics;
+using System.Linq;
+using Content.Shared.CCVar;
+using Content.Shared.Chat.ContentMarkupTags;
+using Content.Shared.Decals;
+using FastAccessors;
+using Robust.Shared.Configuration;
+using Robust.Shared.GameObjects.Components.Localization;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Chat.MarkupTags;
+
+public sealed class EntityNameHeaderContentTag : IContentMarkupTag
+{
+    public string Name => "EntityNameHeader";
+
+    public List<MarkupNode>? OpenerProcessing(MarkupNode node)
+    {
+
+        var list = new List<MarkupNode>();
```

Comment:
dont allocate if u dont use! : D i'm not 100% sure it will be optimized by compiler so better move it down after IF yourself.